.github/ISSUE_TEMPLATE/project-suggestion.md

```diff
@@ -0,0 +1,55 @@
+---
+name: Project Proposal / Suggestion
+about: Idea for a new project that will improve or enhance the TAG Environmental Sustainability (If you want to implement the idea personally, name it "Proposal" if NOT call it "Suggestion").
+title: "[Project Proposal / Suggestion] some descriptive title"
+labels: "triage-required"
+assignees: ''
+---
+
+<!-- Thank you for opening this proposal! If you personally want to lead the project, call it a "proposal", IF NOT, call it a "suggestion". If you want to lead the project, it is much more likely that the proposal will be accepted! -->
+
+## Project details
+
+### Description
+<!-- describe your idea here -->
+
+...
+
+### Impact
+<!-- Describe your hopes for how this would reduce risk for the cloud native ecosystem. Who will this help? How will it help them? -->
+
+...
+
+### Scope
+
```

Comment:
We might add a description as well as an out-of-scope. Also to differentiate from Goals.

Reply:
we got already a section to add a description. I will add a comment to mention out-of-scope things if that helps the project proposal to be more precise.